ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_sorting.py

```diff
@@ -129,3 +129,35 @@ def test_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.lexsort",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_axis=-1,
+        max_axis=0,
+        min_num_dims=1,
+        force_int_axis=True,
+    ),
+    test_with_out=st.just(False),
+)
+def test_numpy_lexsort(
```

Comment:
the test still fails due to this RecursionError: maximum recursion depth exceeded. Do you have any idea why?

Reply:
My idea is the function has no backend implementation and so that I need to implement that in
-ivy/functional/backends/numpy/sorting.py and 
-ivy/functional/ivy/sorting.py 

When I compare with other completed frontend function implementations they didnt implemented backend for their functions because they have implemented frontend functions that have already implemented backend.

When I implemeted also the backend for numpy them the test runs well with numpy but I get an error with the other libraries. Since I did not implement it for other backends, I think that it always gets stuck in the same rejection response in the test and enters an endless loop. Is this a bug that needs to be fixed? or Is this a fixable error in my task?

Reply:
So do you know how or want to implement the backend for all other backends?

Reply:
Actually, it means I am at the beginning of everything. This is what I gave up before. Can't I just make numpy frontend and complete it?

Reply:
I guess you can as someone else can fill in the missing thing later?

Reply:
I think this is the version. For example -just for try- when I add backend implementation for jax, numpy frontend test runs well for jax too without any infinite loop. But without jax backend implementation it goes into infinite loop for jax too

Reply:
that sort of make sense. so maybe add a comment to the code stating that it need all backends filled in? And then fix the easy problems before I maybe just merge this?

Reply:
I added the comment above the frontend test function. Anyone working on it can easily reach it. What are the other easy problems?